### PR TITLE
fix: Accommodating both dictionary and json formatted tag values

### PR DESCRIPTION
**Pull request by alexlokshin-czi**

### CODE REVIEW

File: scripts/happy [hunk 1]
```diff
@@ -59,6 +59,9 @@ class HappyConfig:
             raise CliError(error)
 
     def __getattr__(self, field):
+        print(f'Field: {field}')
```
Comment (jgadling) [inline scripts/happy]: print!

Comment (alexlokshin-czi) [reply scripts/happy]: Arrgh

File: .happy/config.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "terraform_version": "0.13.5",
     "default_env": "rdev",
     "app": "genepi",
-    "default_compose_env": ".env.ecr",
+    "default_compose_env_file": ".env.ecr",
```
Comment (jgadling) [inline .happy/config.json:6]: Why change this parameter name?

**Review by jgadling — APPROVED** [review]

File: .happy/config.json (hunk 2 above)
Comment (alexlokshin-czi) [reply .happy/config.json:6]: default_env (environment name) and default_compose_env (an environment file name) have completely different semantics, we've added some clarity in the golang version.

**Pull request merged**